browser/BUILD.gn

```diff
@@ -107,23 +107,16 @@ source_set("browser_process") {
     ":sparkle_buildflags",
     ":stats_updater",
     ":version_info",
-    "autoplay",
```

Comment:
nit: to reduce the size of this PR can we only sort newly added `#include`'s for legacy .gn files